source/jormungandr/jormungandr/street_network/here.py

```diff
@@ -222,22 +274,67 @@ def _read_response(response, origin, destination, mode, fallback_extremity, requ
 
         return resp
 
-    def get_direct_path_params(self, origin, destination, mode, fallback_extremity):
+    def get_realtime_traffic_parameter(self, request):
+        _realtime_traffic = request.get('_here_realtime_traffic', None)
+        if _realtime_traffic == None:
+            return self.realtime_traffic
+        if (
+            _realtime_traffic != None
```

Comment:
this is not needed as you have already returned `if _realtime_traffic == None` on the line above

Reply:
Finally, I changed the method with a **true/false** value